README.md

```diff
@@ -2,3 +2,303 @@
 
 This repository constains a few React components that are mainly used by
 OrderPlaced and MyOrders to display information about orders.
+
+In order to use the components exported by OrderDetails, you must include in your `manifest.json` file:
+
+```json
+"dependencies": {
+  "vtex.order-details": "1.x",
+},
+```
+
+## Exported components
+
+### Address
+
+This a wrapper for the `<AddressRules/>` component from `vtex.address-form` that renders a formated address. It's props are defined by the interface:
+
+```typescript
+interface Props {
+  address: Address;
+  pickup?: Parcel;
+}
+```
+
+The `Parcel` type comes from `vtex.delivery-packages`, and the `Address` component uses the attribute:
+
+```typescript
+interface Parcel {
+  ...
+  pickupFriendlyName: string | null;
+  ...
+}
+```
+
+### Attachment and Subscription
+
+This `Attachment` component is used for rendering attachments to `OrderItems`. It's props are defined by:
+
+```typescript
+interface Props {
+  attachmentsInfo: Attachment[];
+  bundleInfo: Bundle[];
+  currency: string;
+}
+
+interface Bundle {
+  id: string;
+  attachments: Attachment[];
+  name: string;
+  price: number;
+  quantity: number;
+  imageUrl: string | null;
+  measurementUnit: string;
+  unitMultiplier: number;
+}
+
+interface Attachment {
+  content: any;
+  name: string;
+}
+```
+
+The `Subscription` component is just a customized version to use when the `OrderItem` has an `Attachment` that is a subscription. It's props are different and defined by:
+
+```typescript
+interface Props {
+  attachmentItem: Attachment;
+}
+```
+
+### ButtonLink
+
+This component renders a `Button` or a `ButtonWithIcon` component from `vtex.styleguide` wrapped by a `Link` component from `render-runtime`. It's props are:
+
+```typescript
+interface Props {
+  to: string;
+  icon?: ReactNode;
+  fullWidth?: boolean;
+  openNewWindow?: boolean;
+  variation: string;
+  children: ReactChild;
+}
+```
+
+### CustomerInfo
+
+This a wrapper for the `<ProfileRules />` component from `vtex.profile-form` that renders a formated profile. It's props are defined by the interface:
+
+```typescript
+interface Props {
+  profile: ClientProfile;
+}
+
+interface ClientProfile {
+  email: string;
+  firstName: string | null;
+  lastName: string | null;
+  document: string | null;
+  documentType: string | null;
+  phone: string | null;
+}
+```
+
+### FormattedDate and FormattedPrice
+
+Simple components for rendering formatted dates and prices using `react-intl`. Props for `FormattedDate`:
+
+```typescript
+interface Props {
+  date: string;
+  style: string;
+}
+```
+
+For `FormattedPrice`:
+
+```typescript
+interface Props {
+  value: number;
+  currency: string;
+}
+```
+
+### OrderHeader
+
+Component for rendering basic information on an order visualization, along with `OrderOptions`. It's props are defined by:
+
+```typescript
+interface Props {
+  orderInfo: Order;
+  splitOrder?: boolean;
+  takeaway?: boolean;
```

Comment:
You mean `takeaway`?
This is here because items that are for takeaway always generate new orders into the OrderGroup, so this variable represents the idea that `OrderHeader` is being used to render information on an order that contains takeaway items.

Reply:
It should be clear in the docs how to set this boolean properly.